dockers/docker-orchagent/orchagent.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+# Export platform information. Required to be able to write
+# vendor specific code.
+export platform=`sonic-cfggen -m /etc/sonic/minigraph.xml -v onie_switch_asic`
```

Comment:
We need this change ASAP to be able to run images on MLNX platforms. If it is critical please let me know and I'll change code here and in swss to use **asic_type**.